portal-web/test/functional/com/liferay/portalweb/macros/SearchExperiences.macro

```diff
@@ -0,0 +1,238 @@
+definition {
+
+	macro addBlueprint {
+		Navigator.gotoNavItem(navItem = "Blueprints");
+
+		Click(locator1 = "Button#PLUS");
+
+		PortletEntry.inputTitle(title = "${name}");
+
+		if (isSet(description)) {
+			PortletEntry.inputDescription(description = "${description}");
+		}
+
+		if (isSet(clauseOption)) {
+			LexiconCard.clickCard(card = "Custom Clauses");
+		}
+		else {
+			LexiconCard.clickCard(card = "Liferay Default Clauses");
+		}
+
+		Button.click(button = "Create");
```

Comment:
oh I just realized I misread this macro. The "Create" button creates the blueprint and takes it to the edit form of the blueprint just created. I got it mixed up with the "Save" button in the blueprint edit form which takes the user back to the search exp home page. Though what's odd is how a lot of tests are using the editBlueprint macro immediately after addBlueprint which I think would cause the tests to fail since editBlueprint assumes the user is on the search exp home page.

Reply:
I think it is the same issue as https://github.com/xbrianlee/liferay-portal/pull/605#discussion_r650261000

Reply:
yeah I found out that the tests are actually failing due to the editBlueprint macro call. I'll work on the test fixes and leave the create button within `addBlueprints`.